sdk/common/src/main/java/io/opentelemetry/sdk/resources/Resource.java

```diff
@@ -169,14 +169,20 @@ private static boolean isValidAndNotEmpty(AttributeKey<?> name) {
     return !name.getKey().isEmpty() && isValid(name.getKey());
```

Comment:
Believe we need to add to `ResourceBuilder` too

Reply:
let's do that separately. :)

Reply:
Although, it does get called when the builder is used, just not until you actually try to build the Resources. I guess it might be nicer to fail earlier, but in practice I think it would get caught properly. Please create an issue if you think it should fail earlier when using the builder.

Reply:
I think my comment hit the wrong line - I meant we should add the since tag to that class too not just these methods.

Reply:
oh! sorry about that. Yes, true true. :)